docs/Daos.md

```diff
@@ -0,0 +1,262 @@
+
+What if we want to configure how voting will proceed when someone submits a proposal?
+
+### Creating a new DAO overriding the default voting machine
+
+By default, `DAO.new` configures each scheme with the [AbsoluteVote](api/classes/AbsoluteVoteWrapper) voting machine and default voting parameter values.  The following example retains the default voting machine while overriding its parameters:
+
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  votingMachineParams: {
+    votePerc: 45,
+    ownerVote: false
+  }
+});
+```
+
+Here we supply an alternate voting machine address, supplying it AbsoluteVote parameters:
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  votingMachineParams: {
+    votePerc: 45,
+    ownerVote:false
+    votingMachineAddress: anAddress
+  }
+});
+```
+
+Here we supply a different default voting machine ([GenesisProtocol](api/classes/GenesisProtocolWrapper)), with default voting parameters:
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  votingMachineParams: {
+    votingMachineName: "GenesisProtocol"
+  }
+});
+```
+
+Here we override the default voting machine parameters on just one of the schemes:
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  schemes: [
+    { name: "SchemeRegistrar" },
+    { name: "UpgradeScheme" },
+    {
+      name: "GlobalConstraintRegistrar",
+      votingMachineParams: {
+        votePerc: 30,
+      }
+    }
+  ]
+});
+```
+
+!!! tip
+    If you want change the default voting machine for all calls to `DAO.new` you can do it using the ConfigService setting "defaultVotingMachine". See [Arc.js Configuration Settings](Configuration.md).
+
+!!! note "Important"
+    If you want to use GenesisProtocol on _any_ scheme, you must also add GenesisProtocol as scheme on the DAO itself (see [Creating a new DAO with schemes](#creating-a-new-dao-with-schemes)).
+
+### Creating a new DAO with a non-Universal Controller
+
+There are two types of DAO controller: universal and non-universal.  The default is universal.  You can create a DAO using the DAOstack `Controller` contract by passing `false` for `universalController`:
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  universalController: false
+});
+```
+
+!!! info
+    For more information about choosing between universal and non-universal controllers, see [this article](https://daostack.github.io/arc/contracts/controller/UController/).
+
+### Creating a new DAO with a non-default DaoCreator scheme
+
+In Arc, a DAO is created by the contract called `DaoCreator`.  You can supply your own alternative `DaoCreator` scheme by passing its address in `daoCreatorScheme`:
```

Comment:
Using DAOCreator is not a must to create DAOs.
ARC provide DAOCreator as an helper..

Reply:
Added a link